tasks/22_remove-falsy-values/remove-falsy-values.js

```diff
@@ -0,0 +1,7 @@
+function removeFalsyValues(arr) {
+  arr = arr.filter(function(element) {
```

Comment:
You're overwriting function argument, is it required by the task?

Reply:
no, this is my mistake I will fix it.